backend/apps/core/tests/test_api_views.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+
+
+class TestShoesViewSetGet:
+
+    @pytest.mark.django_db()
+    def test_must_return_success(self, api_client, shoe, shoe_data):
```

Comment:
```suggestion
@pytest.mark.django_db
class TestShoesViewSetGet:

    def test_must_return_success(self, api_client, shoe, shoe_data):
```

Esse decorator pode ser usado diretamente na classe pra evitar essa repetição desnecessária.

Reply:
Essa eu não sabia, gostei vou atualizar aqui!